packages/core/src/child-process.ts

```diff
@@ -71,7 +72,7 @@ export const childProcessStdoutPromise = async (
   return orderedOutput(output).stdout().toString('utf-8').trim()
 }
 
-export const spawnPromise = (
+export const spawnPromise = async (
```

Comment:
redundant `async`

Reply:
The async isn't redundant, if `spawn` logic throw the error is not wrapped.
Ideally, any function that return a Promise should give a strong guarantee that the exception is always wrapped in the promise return value. 
`async` force that, for example:
```
await ((function(){ throw "abc"; return Promise.resolve(5)}()).catch(e=> { }))
```
throws "abc"

```
await ((async function(){ throw "abc"; return Promise.resolve(5)}()).catch(e=> { }))
```
doesn't throw